plugins/defaultplugins/l2plugin/vppcalls/arp_term_vppcalls_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	govppapi "git.fd.io/govpp.git/api"
+	l2ba "github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/bin_api/l2"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/vppcalls/test/impl"
+	"testing"
+	"github.com/ligato/cn-infra/logging/logrus"
+	"github.com/onsi/gomega"
```

Comment:
Change this to `. "github.com/onsi/gomega"` for less code and more readable test cases.

Reply:
ok